proto.mk

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/make -f
+
+###############################################################################
+###                                Protobuf                                 ###
+###############################################################################
+
```

Comment:
This pull request removes `contrib/devtools/Makefile` but I don't think the `contrib` directory is the right place because a `contrib` directory is typically for files not maintained by the core developers of a project.

Reply:
Maybe we can follow something similar to [Akash](https://github.com/ovrclk/akash/tree/master/make).

Reply:
Yea, maybe a `make` directory makes sense.